apps/web/src/design-system/switch/Switch.styles.ts

```diff
@@ -1,5 +1,7 @@
 import { createStyles, MantineTheme } from '@mantine/core';
 
+import { colors } from '../config';
+
 export default createStyles((theme: MantineTheme, _params, getRef) => {
```

Comment:
fixed the switch styles here, a lot of style things have changed, so I've made it look the same like it was before